bazel/linux/runner.bzl

```diff
@@ -195,6 +201,13 @@ def create_runner(ctx, archs, code, runfiles = None, extra = {}):
             ),
         )
 
+    if ki.arch == "aarch64":
```

Comment:
just a dumb note, nothing to do --- the compiler and bazel both use `aarch64`, but the Linux kernel uses `arm64`.  it so dumb these are different in different places...  i think what you have here is correct.  when writing anything bazel use `aarch64`.